include/variables.h

```diff
@@ -164,12 +156,10 @@ extern s16 gSpoilingItems[3];
 extern s16 gSpoilingItemReverts[3];
 extern FlexSkeletonHeader* gPlayerSkelHeaders[2];
 extern u8 gPlayerModelTypes[][5];
-extern Gfx* D_80125D68[];
 extern Gfx* D_80125DE8[];
-extern Gfx* D_80125E08[];
+extern Gfx* gLeftHandDLs[];
 extern Gfx* D_80125E18[];
-extern Gfx* D_80125E58[];
-extern Gfx* D_80125EF8[];
+extern Gfx* gBoomerrangDLs[];
```

Comment:
we should probably prefix these arrays with player or something, seems easy to have global variable name clashes with something as generic as "left hand dlists"